### PR TITLE
Replace social media widgets with social media links

### PR DESCRIPTION
**Pull request by radusuciu**
Removing social media widgets which are dynamically loaded has two main benefits:
- faster page loads
- better for privacy

The main difference is instead of clicking "Tweet" and being invited to tweet, you are linked to the Mixxx Twitter page (as requested in #13). In my opinion this is likely to increase engagement with Mixxx's social media. Would be nice if there was some data on how much use the widgets are getting.

The social media links in this PR are styled very similar to the widgets. Also removed jQuery as the only uses of Javascript on the website were simple enough to express without jQuery. It can always be added when needed.

### CODE REVIEW

**Comment by ronso0** [comment]
I just now became really aware of the website repo..

I wanted to have a quick look at the icons (maybe apply Mixxx style?) but noticed the svgs are inline. does it have any benefit? do you mind storing the icons in separate files so they're easier to review and edit?

**Comment by ronso0** [comment]
okay, got it up & running.
before I never was aware of those buttons as my browser blocks fb items..
it's nice to get rid of tracking objects and just link to the social sites instead!

as I thought in the first place, we could omit the text and just use icons.
Without text the buttons would also be small enough to fit underneath **Manual**, next to **Press**. This would make the site look a bit tidier than now.

**Comment by radusuciu** [comment]
Svg as files would provide a minor benefit through caching I suppose, but it may be balanced out by increased number of requests (unless on HTTP2) - I think. I don't recall exactly but I think I just did it because it was easier.. can of course convert to files

I can pick this back up at some point but I'm currently writing my PhD Thesis so... lol. I'll be done in May but feel free to take over if you want!

Also, I should mention that I was thinking of splitting this up into two pull requests since I also cleaned up some JS.

**Comment by Be-ing** [comment]
I just remembered about this PR. Thank you for reducing external dependencies and getting rid of the tracking. I noticed that the icons can overlap the "DJ Your Way" text:
image
compared to the current website:
image

How about getting rid of the "Twitter" and "Facebook" text and just using their icons?

**Comment by Be-ing** [comment]
Also thanks for adding the Zulip button :+1:

**Comment by ronso0** [comment]
yeah, that's pretty much what I proposed above and that would help in narrow view, too
> Without text the buttons would also be small enough to fit underneath Manual, next to Press. This would make the site look a bit tidier than now.

@radusuciu Are able to finish this or should someelse take over?

File: static/js/mixxx.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+var OSName = getOSName();
+
+function getOSName() {
+  var OSName="your OS";
+  if (navigator.appVersion.indexOf("Win")!=-1) OSName="Windows";
```
Comment (Be-ing) [inline static/js/mixxx.js]: Let's use the same style we use for JS in controller mappings:
```suggestion
  if (navigator.appVersion.indexOf("Win") !== -1) {
    OSName="Windows";
  }
```

**Review by Be-ing — CHANGES_REQUESTED** [review]

**Comment by ronso0** [comment]
yeah, that's pretty much what I proposed above and that would help in narrow view, too
> Without text the buttons would also be small enough to fit underneath Manual, next to Press. This would make the site look a bit tidier than now.

@radusuciu Are able to finish this or should someelse take over?

**Comment by radusuciu** [comment]
@ronso0 @Be-ing I can take into account above suggestions and amend the PR!

**Comment by radusuciu** [comment]
Reverted a commit that added these new side link icons to all pages, as some pages already had elements in the same position, and I think that that would warrant a separate PR.

**Comment by radusuciu** [comment]
Other changes:
- moved inline svg to separate files
- re-organized js a bit. Namespaced what few functions there are and extracted some js from the download page to it's only file

**Comment by ronso0** [comment]
Looks good on desktop and mobile view, works as expected.
Are there any corner cases to test?

**Comment by radusuciu** [comment]
> Looks good on desktop and mobile view, works as expected.
> Are there any corner cases to test?

Not that I can think of other than trying at different screen sizes.

**Comment by Be-ing** [comment]
Thanks for taking care of this.
